pyproject.toml

```diff
@@ -10,6 +10,7 @@ dependencies = [
     "fastapi>=0.118.0",
     "psycopg2-binary>=2.9.11",
     "pydantic>=2.11.10",
+    "pytest>=8.4.2",
```

Comment:
Do not add pytest as a runtime dependency, it should be a dev dependency only (`uv add --dev pytest`)